scripts/hid_configurator/README.rst

```diff
@@ -33,6 +33,15 @@ For more information about the architecture of the nRF Desktop configuration cha
 The configuration channel uses a cross-platform HIDAPI library to communicate with HID devices attached to the operating system.
 See `Dependencies`_ for more information.
 
+.. note::
+   Since NCS 2.7 release, format version of the :file:`dfu_application.zip` file generated by the build system was updated from ``0`` to ``1``.
```

Comment:
```suggestion
   Since the |NCS| v2.7.0 release, the format version of the :file:`dfu_application.zip` file generated by the build system was updated from ``0`` to ``1``.
```